index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <div>
+        <i class="fas fa-bars"></i>
+    </div>
```

Comment:
- [ ] Great job implementing the `the headline section (section right after the header):` :+1: , But I think there are some improvements you can make here before moving to the next milestone. :+1: 
- [ ] Following the best `HTML practices`, you should wrap this entire section with the `section semantic tag.` +1 This is great as it follows the best industry standard. :smile: :tada: